_docs/tooling/maven.md

```diff
@@ -91,5 +92,20 @@ Here is the list of properties supported by the goal:
     increase compilation time, sometimes can make code even slower; 
     recommended for WebAssembly.
 * **targetType** &ndash; what code to generate. Following options are supported:
-  * *JAVASCRIPT*
-  * *WEBASSEMBLY*
\ No newline at end of file
+  * *JAVASCRIPT* to generate JavaScript
+  * *WEBASSEMBLY* to generate WebAssembly.
+  
+  The default value is JAVASCRIPT.
+* **stopOnErrors** The default value is `true`.
+* **maxTopLevelNames** The default value is 10,000
+* **fastGlobalAnalysis**
```

Comment:
What is the purpose in just enumerating available options without any explanation?

Reply:
Simply help the person who later fills in the gaps in the documentation.